pkg/config/config.go

```diff
@@ -305,6 +305,11 @@ func (nsxVersion *NsxVersion) getVersion(host string, userName string, password
 		log.Error(err, "failed to get nsx version")
 		return err
 	}
+	if resp.StatusCode > 300 {
```

Comment:
Do you think we should check whether the status code is exact 200 or 201, to detect if it's a failure?

Reply:
ok